azure-pipelines.yml

```
@@ -35,8 +36,11 @@ jobs:
   - script: python3 -m pip install poetry && poetry install
     displayName: 'Install Project Environment'
 
-  - script: flake8
-    displayName: 'Lint flake8-annotations'
+    # This will also take care of linting, as we utilize a flake8 hook
+    # Only run this in one python environment
+  - script: pre-commit run --all-files
+    condition: eq(variables['python.version'], '3.8')
```

Pretty sure `succeeded()` is not implicit when specifying custom conditions.

```suggestion
    condition: eq(succeeded(), variables['python.version'], '3.8')
```

---

Dropped this in with the cache change